sda/internal/config/config.go

```diff
@@ -192,29 +192,29 @@ func NewConfig(app string) (*Config, error) {
 	if viper.IsSet("configFile") {
 		viper.SetConfigFile(viper.GetString("configFile"))
 	}
-	log.Infoln("reading config")
+	log.Debugf("Attempting to read configuration...")
 	if err := viper.ReadInConfig(); err != nil {
-		log.Infoln(err.Error())
+		log.Debugf("Error reading config : %v", err.Error())
```

Comment:
```suggestion
		log.Debugf("Error reading config : %s", err.Error())
```

`err.Error()` is by definition a string.

The alternative would be:

```
		log.Debugf("Error reading config : %v", err)
```